### ISSUE TITLE
sync issues with the android client via dropbox or nextcloud, just the same

### ISSUE THREAD

**Issue by stepfister** [laurent22/joplin#964]
### Operating system

- Android

### Application

- Mobile

I tried to sync my notes on the android app. It works only the first time after a fresh installation. Further changes in the notes via the desktop app on linux or on the mobile app on ios are not syncing with the android device. I'm using linux mint sarah on the desktop, ios 12 on the ipadpro and android in version 6. marshmellow on my phone. I tried as sync-method dropbox. It works perfect with linux and ios, but  without success on android. Today I installed nextcloud and I'm syncing sucessfully via my desktop linux and with the ios app. But android is not working either. I can delete Joplin on my phone, install it new and then it will sync one time but not more often. Should I buy an apple iphone?

The cause of this problem was the formerr use of encryption. After disabling encryption the whole problem with android was there. The master key disturbs the behavior of the android app. Can't get rid of him.

**Comment by stepfister** [comment]
Ok, I tried some ways. Used DB Browser Sqlite to delete the master key. It comes back. Found the information in this thread of the [forum](https://discourse.joplin.cozic.net/t/android-out-of-sync/644/15). It is them same issue with me. I used encryption but disabled it and then there was the syncing problem with android. To make a long story short. The only solution to get rid of this encryption mess was

1.) export of all my notes in the desktop app to *.jex.
2.) delete all mobile apps (ios, android in my case)
3.) delete all files of joplin in my nextcloud-folder
4.) Sync of the desktop app with my nextcloud -> deletes all of the notes
5.) restore backup of the jex-Archive in the desktop-app
6.) Let it sync all the notes with nextcloud
7.) Installed the mobile apps back -> and syncing again

This was the only way for me to get rid of this strange problem. The export to *.jex removes all the encryption and the master key.

I think there should be a better solution for this issue. I fought several weeks with this problem.
